tests/integration/pkg/jwt/auth.go

```diff
@@ -25,6 +26,11 @@ func GetAccessToken(oauth2Cfg *clientcredentials.Config, config *Config) (string
 		Jar:     jar,
 	}
 
+	if len(tokenType) > 0 {
+		oauth2Cfg.EndpointParams = make(url.Values)
```

Comment:
This is a write on a shared instance of oauth2Cfg and since we run tests in parallel this can cause issues.
There are multiple solutions for this issue, e.g. passing the conf by value to this function or not having the conf as a pointer at all.

Reply:
https://github.com/kyma-project/api-gateway/pull/205/commits/17192a183f20aa8bef3097a143d56634203ea6fc